activemodel/test/cases/helper.rb

```diff
@@ -14,10 +14,6 @@
 
 require 'mocha/setup' # FIXME: stop using mocha
 
-# Skips the current run on Rubinius using Minitest::Assertions#skip
-def rubinius_skip(message = '')
-  skip message if RUBY_ENGINE == 'rbx'
-end
```

Comment:
I'm not sure we should remove the helper, it seems to be defined in all other abstract tests we have for rails, even though they might be used in some of the frameworks. The same for jruby_skip for example. So either we consider these available on all tests, or we remove them on the parts they are not used as well. Wdyt?